docs/deployment_guide/partner_editable/_settings.adoc

```diff
@@ -1,11 +1,38 @@
-:partner-solution-project-name: cfn-ps-ammos-smallsat-toolkit
-:partner-solution-github-org: aws-ia
+// Change the following attributes.
+:quickstart-project-name: quickstart-ammos-smallsat-toolkit
```

Comment:
These docs changes need to come from.. main.. I think. I suggest we do doc updates separately so that we don't end up with more clutter in this PR than necessary.